sass/style.css

```diff
@@ -48,11 +48,15 @@ a {
   color: white;
 }
 
-.search-filter-container {
-  position: sticky;
-  top: 58px;
-  z-index: 1;
-  background: white;
+@media screen and ( min-width: 992px ) and ( min-height: 768px )
```

Comment:
Just tested in full screen mode, looks good but can you loosen the min-height a bit so more desktop users (esp ones with lower resolutions) get this effect?

Reply:
From GA,

Here are most common desktop resolutions (window height will be smaller):
1920x1080
1440x900
1366x768
1536x864
1280x800

Here are mobile resolutions:
375x812
414x896
375x667
375x667

Reply:
So looks like you can be more aggressive on the width side and less on the height?